rspace/src/test/scala/coop/rchain/rspace/StorageActionsTests.scala

```diff
@@ -377,6 +404,9 @@ trait StorageActionsTests extends StorageTestsBase[String, Pattern, String, Stri
 
     store.eventsCounter.getProducesCount shouldBe 3
     store.eventsCounter.getConsumesCount shouldBe 3
+    store.eventsCounter.getProducesCommCount shouldBe 0
+    store.eventsCounter.getConsumesCommCount shouldBe 3
+    store.eventsCounter.getInstallCommCount shouldBe 0
```

Comment:
Isn't this the same as [this test](https://github.com/rchain/rchain/pull/914/files#diff-ef153c9df8e794c30aa19a89654b8185R189) ?

Reply:
See the explanation above.